etcdserver/server.go

```diff
@@ -83,6 +83,9 @@ const (
 
 	// maxPendingRevokes is the maximum number of outstanding expired lease revocations.
 	maxPendingRevokes = 16
+
+	defaultMaxRequestBytes     = 1.5 * 1024 * 1024
```

Comment:
probably set this using `embed.NewConfig(...)` and lose the `cfg.MaxRequestBytes <= 0` check